src/sentry/lang/javascript/processor.py

```diff
@@ -470,14 +472,19 @@ def fetch_release_archive_for_url(release, dist, url) -> Optional[IO]:
             return None
         else:
             try:
-                file_ = fetch_retry_policy(lambda: ReleaseFile.cache.getfile(releasefile))
+                with sentry_sdk.start_span(op="fetch_release_archive_for_url.fetch_releasefile"):
+                    file_ = fetch_retry_policy(lambda: ReleaseFile.cache.getfile(releasefile))
             except Exception:
                 logger.error("sourcemaps.read_archive_failed", exc_info=sys.exc_info())
 
                 return None
 
-            # This will implicitly skip too large payloads.
-            cache.set(cache_key, file_.read(), 3600)
+            with sentry_sdk.start_span(op="fetch_release_archive_for_url.read_for_caching"):
+                contents = file_.read()
+            with sentry_sdk.start_span(op="fetch_release_archive_for_url.write_to_cache"):
```

Comment:
Perhaps would be helpful to add the size of "contents" to the span as additional data, to check if there's correlation between size and time.

Reply:
I think this makes sense.. probably should do this for every span. Keep in mind we don't have access to local vars in performance product